src/in-house-liquidity/index.test.ts

```diff
@@ -54,4 +65,217 @@ describe('In House Liquidity Calls', () => {
       ).toThrow('Cannot call IHL because walletAddress is null')
     })
   })
+
+  describe('makeRequest', () => {
+    it('makes a FC authenticated call', async () => {
+      ;(getAuthHeaders as jest.Mock).mockResolvedValueOnce({ 'header-key': 'header-val' })
+      mockFetch.mockResponseOnce(JSON.stringify({}), { status: 201 })
+      await makeRequest({
+        providerInfo: mockProviderInfo,
+        path: '/some/path',
+        options: {
+          method: 'POST',
+          body: { some: 'data' },
+        },
+      })
+
+      expect(getAuthHeaders).toHaveBeenCalledWith({ providerInfo: mockProviderInfo })
+      expect(mockFetch).toHaveBeenCalledWith(`${networkConfig.inHouseLiquidityURL}/some/path`, {
+        headers: {
+          'Content-Type': 'application/json',
+          'header-key': 'header-val',
+        },
+        method: 'POST',
+        body: { some: 'data' },
+      })
+    })
+    it('makes a FC unauthenticated call', async () => {
+      mockFetch.mockResponseOnce(JSON.stringify({}), { status: 201 })
+      await makeRequest({
+        path: '/some/path',
+        options: {
+          method: 'GET',
+        },
+      })
+      expect(mockFetch).toHaveBeenCalledWith(`${networkConfig.inHouseLiquidityURL}/some/path`, {
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        method: 'GET',
+      })
+    })
+  })
+
+  describe('getAuthHeaders', () => {
+    let mockFiatConnectClient: jest.Mocked<FiatConnectApiClient>
+
+    beforeEach(async () => {
+      mockFiatConnectClient = (await getFiatConnectClient(
```

Comment:
any reason to call getFiatConnectClient here? can't we just use the mock functions we define above as global vars?

Reply:
I thought that calls to jest.mock were hoisted so you couldn't reference global vars within them?

Reply:
I think this works if global variables are referenced inside another jest.fn(). E.g., https://github.com/valora-inc/wallet/pull/2855/files#diff-7caff590ac25eb7fe43d613aa98c7e1b1d4e472211ab028f2aa17b11b2514bc3R39